src/DynamoCoreWpf/Views/Search/LibraryView.xaml

```diff
@@ -195,25 +195,67 @@
-                           VerticalAlignment="Center"
-                           Name="CategoryName">
-                    <TextBlock.Style>
-                        <Style>
-                            <Style.Triggers>
-                                <DataTrigger Binding="{Binding IsExpanded, 
+
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock Foreground="{StaticResource NestedMemberTextColor}"
+                               FontSize="13"
+                               Text="{Binding Path=Name}"
+                               VerticalAlignment="Center"
+                               Name="CategoryName"
+                               Grid.Column="0">
+                        <TextBlock.Style>
+                            <Style>
+                                <Style.Triggers>
+                                    <DataTrigger Binding="{Binding IsExpanded, 
                                                                RelativeSource={RelativeSource FindAncestor, 
                                                                                               AncestorType={x:Type TreeViewItem}}}"
-                                             Value="True">
-                                    <Setter Property="TextBlock.FontWeight"
-                                            Value="SemiBold" />
-                                </DataTrigger>
-                            </Style.Triggers>
-                        </Style>
-                    </TextBlock.Style>
-                </TextBlock>
+                                                 Value="True">
+                                        <Setter Property="TextBlock.FontWeight"
+                                                Value="SemiBold" />
+                                    </DataTrigger>
+                                </Style.Triggers>
+                            </Style>
+                        </TextBlock.Style>
+                    </TextBlock>
+                    <TextBlock x:Name="ElementTypeText"
+                               Grid.Column="1"
+                               Text="{Binding Path=ElementType,Converter={StaticResource ElementTypeToShortConverter}}"
+                               VerticalAlignment="Center"
+                               HorizontalAlignment="Right"
+                               TextBlock.FontSize="10"
+                               TextBlock.Foreground="{StaticResource NestedMemberTextColor}"
+                               Margin="0,0,10,0">
+                        <TextBlock.Style>
+                            <Style TargetType="{x:Type TextBlock}">
+                                <Setter Property="Visibility"
+                                        Value="Hidden" />
```

Comment:
I think you can fold `Visibility="Hidden"` into the `TextBlock` itself without this additional `Style.Setter`.